src/scss/components/_thumbnail.scss

```diff
@@ -57,9 +60,31 @@
 
   &__photo {
     width: 100%;
+    overflow: hidden;
+    border-radius: $borderRadius;
+    display: flex;
+    justify-content: center;
+    height: 170px;
+
+    img {
+      height: 170px;
```

Comment:
Can we use `height: 100%` here? At [line 67](https://github.com/getninjas/gaiden/pull/72/files#diff-cf47783a2c84bc5997de9bb31a11794bR67), we have this number already.

This is the same motivation that the [line 87] exists (https://github.com/getninjas/gaiden/pull/72/files#diff-cf47783a2c84bc5997de9bb31a11794bR87).